account_fiscal_year_closing/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Empty line can be removed and license terms ?

Reply:
Thank you. I would ask @pedrobaeza what does he think, not a specialist in these matters myself.

Reply:
No Problem :+1: